crates/datastore/src/system_tables.rs

```diff
@@ -927,6 +960,18 @@ pub struct StClientRow {
     pub connection_id: ConnectionIdViaU128,
 }
 
+/// System table [ST_CONNECTION_CREDENTIALS_NAME]
+///
+/// | connection_id                      | jwt_payload                                             |
+/// |------------------------------------|---------------------------------------------------------|
+/// | 0x6bdea3ab517f5857dc9b1b5fe99e1b14 | '{"iss":"issuer","sub":"user-id","iat":1629212345,...}' |
+#[derive(Clone, Debug, Eq, PartialEq, SpacetimeType)]
+#[sats(crate = spacetimedb_lib)]
+pub struct StConnectionCredentialsRow {
+    pub connection_id: ConnectionIdViaU128,
```

Comment:
We don't otherwise enforce uniqueness of connection IDs, only  of `(ConnectionId, Identity)` pairs.